server/request/request.go

```diff
@@ -370,22 +376,29 @@ func getMetadataFromToken(token string) (string, bool, string) {
 		// this is allowed for m2m apps
 	}
 
+	role, err = jsonparser.GetString(decodedToken, JWTTigrisClaimSpace, Role)
+	if err != nil {
+		log.Debug().Err(err).Msg("Could not read user role")
+		// this is allowed for transition
+		// TODO: this should be disabled once RBAC is rolled out
+	}
+
 	sub, err := jsonparser.GetString(decodedToken, Subject)
 	if err != nil {
 		log.Error().Err(err).Msg("Could not read subject")
-		return defaults.UnknownValue, false, ""
+		return defaults.UnknownValue, false, "", ""
 	}
-	return namespaceCode, len(userEmail) > 0, sub
+	return namespaceCode, len(userEmail) > 0, sub, role
 }
 
-func GetMetadataFromHeader(ctx context.Context) (string, bool, string) {
+func GetMetadataFromHeader(ctx context.Context) (string, bool, string, string) {
```

Comment:
Can we add a comment about what the method is returning? The last two strings may be confusing.

Reply:
good point. added comment.